build.sbt

```diff
@@ -4,6 +4,16 @@ organization in ThisBuild := "me.shadaj"
 
 scalaVersion in ThisBuild := "2.12.8"
 
+Compile / scalaSource := baseDirectory.value / "src" /  "main" / "scala"
```

Comment:
While you're here... pls also provide the list of required software/C headers (`re2` etc.) to install before `sbt test:compile` in README.md

Reply:
I suppose ensuring setup for scala native (including optional libraries)  is obligatory, I don't know which exactly but it's safe to say all
```
And following runtime library dependencies:
libunwind 0.99 or newer (built-in on macOS)
Boehm GC 7.6.0 (optional)
Re2 2017-01-01 (optional)
zlib 1.2.8 or newer (optional)
```
From https://scala-native.readthedocs.io/en/v0.3.9-docs/user/setup.html

Reply:
Ok! pls add to README so that it's not lost

Reply:
(at least the link to scala-native setup)

Reply:
ok